app/Planner.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Planner extends Model
+{
+    protected $fillable = ['name', 'start_date', 'end_date', 'genre_1', 'genre_2', 'genre_3', 'genre_4', 'genre_5', 'headliner_1', 'headliner_2', 'headliner_3', 'headliner_4', 'headliner_5', 'description', 'todo_list', 'playlist', 'planner_image'];
```

Comment:
Yes, i think it would be better to create separate tables for headliners and genres.